src/Discord.Net.Interactions/InteractionModuleBase.cs

```diff
@@ -44,57 +44,57 @@ internal void SetContext(IInteractionContext context)
         protected virtual Task DeferAsync(bool ephemeral = false, RequestOptions options = null)
             => Context.Interaction.DeferAsync(ephemeral, options);
 
-        /// <inheritdoc cref="IDiscordInteraction.RespondAsync(string, Embed[], bool, bool, AllowedMentions, MessageComponent, Embed, RequestOptions)"/>
+        /// <inheritdoc cref="IDiscordInteraction.RespondAsync(string, Embed[], bool, bool, AllowedMentions, MessageComponent, Embed, RequestOptions, PollProperties)"/>
         protected virtual Task RespondAsync(string text = null, Embed[] embeds = null, bool isTTS = false, bool ephemeral = false,
             AllowedMentions allowedMentions = null, RequestOptions options = null, MessageComponent components = null, Embed embed = null)
```

Comment:
Probably forgot to add `, PollProperties poll = null` for `Respond*` interaction functions?

Reply:
oops
yeah just forgot about interaction framework